index.js

```diff
@@ -452,11 +452,16 @@ function processEvent(event, serviceName, streamName, callback) {
 
 			processFinalRecords(records, streamName, destinationStream, asyncCallback);
 		    }, function(err, results) {
-			results.map(function(item) {
-			    console.log(JSON.stringify(item));
-			});
-
-			callback();
+			if (err) {
+				callback(err);
+			} else {
+				if (debug) {
```

Comment:
Please let me know if this change is something I should remove from this commit.